rpc/helpers.go

```diff
@@ -89,7 +89,16 @@ func (h *Handler) blockHeaderByID(id *BlockID) (*core.Header, *jsonrpc.Error) {
 
 func adaptExecutionResources(resources *core.ExecutionResources, v0_6Response bool) *ExecutionResources {
 	if resources == nil {
-		return &ExecutionResources{}
+		if !v0_6Response {
+			return &ExecutionResources{
+				DataAvailability: &DataAvailability{
+					L1Gas:     uint64(0),
+					L1DataGas: uint64(0),
```

Comment:
Let's use just `&DataAvailability{}` you don't need explicitly set these values to zero.
Here is good talk about this - https://www.youtube.com/watch?v=PAAkCSZUG1c&t=385s&ab_channel=TheGoProgrammingLanguage